packages/expo/ios/AppDelegates/EXReactNativeFactoryDelegate.mm

```diff
@@ -22,12 +21,6 @@ - (Class)getModuleClassFromName:(const char *)name
 #endif
 }
 
-- (std::shared_ptr<facebook::react::TurboModule>)getTurboModule:(const std::string &)name
-                                                      jsInvoker:(std::shared_ptr<facebook::react::CallInvoker>)jsInvoker
-{
-  return facebook::react::DefaultTurboModules::getTurboModule(name, jsInvoker);
-}
-
```

Comment:
not necessarily to override this because it's exactly the same as DefaultReactNativeFactoryDelegate